virtcontainers/kata_agent.go

```diff
@@ -512,6 +528,44 @@ func (k *kataAgent) startProxy(sandbox *Sandbox) error {
 	return nil
 }
 
+func (k *kataAgent) getAgentUrl() (string, error) {
+	return k.agentURL()
+}
+
+func (k *kataAgent) setProxy(sandbox *Sandbox, proxy proxy, pid int, url string) error {
+	if url == "" {
+		var err error
+		if url, err = k.agentURL(); err != nil {
+			return err
+		}
+	}
+
+	logger := k.Logger().WithFields(logrus.Fields{
+		"proxy-pid": pid,
+		"proxy-url": url,
+	})
+	if sandbox != nil {
+		logger = logger.WithField("sandbox", sandbox.id)
+	}
+	logger.Infof("set proxy")
```

Comment:
I'm not sure what this is for? If we need it, it can be `logger.Info()` too. Might be better to drop it (and potentially add some trace spans in these files instead :)

Reply:
ok, I'll drop it.